### PR TITLE
Fix an exception on EXEC variable name

### LINKED ISSUE #1077: Variable definition 01 EXEC produces NPE
## Description ##

Variable definition 01 EXEC is recognized as the END PROGRAM statement and produces NPE

## Steps to reproduce ##
1. Add a variable definition 01 EXEC
2. See server freezes

## Expected behavior ##

The document is parsed as expected.

### PR DESCRIPTION
**Pull request by temanbrcom**
Fix #1077

### CODE REVIEW

File: server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java [hunk 1]
```diff
@@ -387,30 +387,34 @@ private void checkFileControlUniqueness(FileControlEntryContext ctx, String file
...
-                        it, code.getTokenStream(), code.getTree(), EmbeddedCodeNode.Language.SQL)))
+    return processEmbeddedNodes(ctx, ctx.sqlCode(), EmbeddedCodeNode.Language.SQL);
+  }
+
+  private List<Node> processEmbeddedNodes(ParserRuleContext parent,
+      ParserRuleContext ctx, EmbeddedCodeNode.Language language) {
+    return ofNullable(ctx)
+        .map(ParserRuleContext::getStart)
+        .map(embeddedCodeParts::get)
+        .map(
```
Comment (grianbrcom) [inline server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java:404]: Try to use `peek` here.

**Review by grianbrcom — APPROVED** [review]
:ear_of_rice:

File: server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java (hunk 1 above)
Comment (temanbrcom) [reply server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java:404]: The intention of `peek` is debugging, so we will have a code smell here.

Comment (grianbrcom) [reply server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java:404]: OK then.

Comment (temanbrcom) [reply server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java]: Still, it looks dirty, I will redo it.

**Review by grianbrcom — APPROVED** [review]
:wolf:

**Pull request merged**